templates/experiment.html

```diff
@@ -1,12 +1,50 @@
-<div class="card info-card">
+<!-- BEGIN INVALID AMP - don't try at home -->
+<div id="experiment-warning" class="card info-card">
   <h4>Experimental Mode</h4>
-  <p>The <code>{{component}}</code> component is currently in experimental mode and needs to be manually enabled:</p>
-
-  <ol>
-    <li>check the box under <a href="https://cdn.ampproject.org/experiments.html">CDN experiments</a>.
-    </li>
-    <li>enter the following line in the <a href="https://developer.chrome.com/devtools/docs/console">Javascript console</a>:
-      <pre><code>AMP.toggleExperiment('{{component}}')</code></pre>
-    </li>
-  </ol>
+
+  <script>
+(function() {
+  updateExperimentWarning();
+  showInvalidAmpMessage();
+})();
+
+function updateExperimentWarning() {
+  if (!isExperimentOn('{{component}}')) {
+    return;
+  }
+  const experimentWarning = document.getElementById('experiment-warning');
+  experimentWarning.style.display = 'none';
+}
+
+/* 
+ * Not using AMP.isExperimentOn(...) as we don't know when the AMP runtime is
```

Comment:
You could use our super undocumented API to do this. Might be better than using our undocumented cookie API :)

```js
window.AMP = window.AMP || []).push(function(AMP) {
  // Code that can use AMP goes here.
});
```